core/notifications/src/email_executor/smtp/mod.rs

```diff
@@ -0,0 +1,39 @@
+mod config;
+pub mod error;
+
+use lettre::{
+    message::{Mailbox, Message},
+    transport::smtp::authentication::Credentials,
+    AsyncSmtpTransport, AsyncTransport, Tokio1Executor,
+};
+
+use crate::messages::LocalizedMessage;
+
+pub use config::*;
+use error::*;
+
+#[derive(Clone)]
+pub struct SmtpClient {
+    client: AsyncSmtpTransport<Tokio1Executor>,
+}
+
+impl SmtpClient {
+    pub fn init(config: SmtpConfig) -> Result<Self, SmtpError> {
+        let creds = Credentials::new(config.username, config.password);
+        let client: AsyncSmtpTransport<Tokio1Executor> =
+            AsyncSmtpTransport::<Tokio1Executor>::starttls_relay("smtp.gmail.com")?
```

Comment:
Why is this endpoint hard coded?

Reply:
was just for testing should come in via the config.